web-api/src/error/common.rs

```diff
@@ -22,101 +22,99 @@ use derive_more::From;
-
-/// The requested property was not found.
-#[derive(Debug, Error, Display, Serialize)]
-pub(crate) struct PropertyNotFound;
-
-impl_application_error!(PropertyNotFound => BAD_REQUEST);
+impl_application_error!(DocumentPropertyNotFound => BAD_REQUEST, INFO);
 
 /// Malformed user id.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct InvalidUserId {
     pub(crate) value: String,
 }
 
-impl_application_error!(InvalidUserId => BAD_REQUEST);
+impl_application_error!(InvalidUserId => BAD_REQUEST, INFO);
 
 /// Malformed document id.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct InvalidDocumentId {
     pub(crate) value: String,
 }
 
-impl_application_error!(InvalidDocumentId => BAD_REQUEST);
+impl_application_error!(InvalidDocumentId => BAD_REQUEST, INFO);
 
 /// Malformed document property id.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct InvalidDocumentPropertyId {
     pub(crate) value: String,
 }
 
-impl_application_error!(InvalidDocumentPropertyId => BAD_REQUEST);
+impl_application_error!(InvalidDocumentPropertyId => BAD_REQUEST, INFO);
 
 /// Malformed document tag.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct InvalidDocumentTag {
     pub(crate) value: String,
 }
 
-impl_application_error!(InvalidDocumentTag => BAD_REQUEST);
+impl_application_error!(InvalidDocumentTag => BAD_REQUEST, INFO);
 
 /// Failed to delete some documents.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct FailedToDeleteSomeDocuments {
     pub(crate) errors: Vec<DocumentIdAsObject>,
 }
 
-impl_application_error!(FailedToDeleteSomeDocuments => BAD_REQUEST);
+impl_application_error!(FailedToDeleteSomeDocuments => BAD_REQUEST, INFO);
 
 /// The ingestion of some documents failed.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct IngestingDocumentsFailed {
     pub(crate) documents: Vec<DocumentIdAsObject>,
 }
 
-impl_application_error!(IngestingDocumentsFailed => INTERNAL_SERVER_ERROR);
+impl_application_error!(IngestingDocumentsFailed => INTERNAL_SERVER_ERROR, ERROR);
 
 /// Failed to set some document candidates.
 #[derive(Debug, Display, Error, Serialize)]
 pub(crate) struct FailedToSetSomeDocumentCandidates {
     pub(crate) documents: Vec<DocumentIdAsObject>,
 }
 
-impl_application_error!(FailedToSetSomeDocumentCandidates => BAD_REQUEST);
+impl_application_error!(FailedToSetSomeDocumentCandidates => BAD_REQUEST, INFO);
 
 /// The history does not contains enough information.
 #[derive(Debug, Error, Display, Serialize)]
 pub(crate) struct HistoryTooSmall;
 
-impl_application_error!(HistoryTooSmall => BAD_REQUEST);
+impl_application_error!(HistoryTooSmall => BAD_REQUEST, INFO);
 
 /// Custom error for 400 Bad Request status code.
 #[derive(Debug, Error, Display, Serialize, From)]
 pub(crate) struct BadRequest {
     pub(crate) message: Cow<'static, str>,
 }
 
-impl_application_error!(BadRequest => BAD_REQUEST);
+impl_application_error!(BadRequest => BAD_REQUEST, INFO);
```

Comment:
:question:  is info the right choice for this kind of wrong API usage errors, normally you would log them at a debug level

Reply:
just discussed this with @acrrd , we decided to keep it at `info` level for better visibility in case some customer uses the api in a wrong way again.